package.json

```diff
@@ -109,6 +107,7 @@
     "eslint": "8.57.0",
     "eslint-plugin-async-await": "0.0.0",
     "eslint-plugin-import": "2.29.1",
+    "fs-extra": "11.2.0",
```

Comment:
It's still used in tests, I didn't want to rewrite all of them. It's fine as a dev dependency

Reply:
that's fair, it seems to be used only in the test.chromium.js unit test, but it is not a big deal if we keep it as a dev dependency in the meantime. Let's keep that test as is :+1: